fluid/DeepASR/model.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import paddle.v2 as paddle
+import paddle.v2.fluid as fluid
+
+
+def stacked_lstmp_model(hidden_dim,
+                        proj_dim,
+                        stacked_num,
+                        parallel=False,
+                        is_train=True,
+                        class_num=1749):
```

Comment:
I think we should not give a default value for `class_num`

Reply:
Done